repository/MonticelloFileTree-Core.package/MCFileTreeRepository.class/instance/readerClassForFileNamed..st

```diff
@@ -0,0 +1,4 @@
+actions
+readerClassForFileNamed: fileName
+	^ "MCReader readerClassForFileNamed: fileName"
+		MCFileTreeReader
```

Comment:
But this was a change to MCFileTreeRepository (which is why I called it out) ... agree with using `cypress://` since the format is completely different ... I will have to teach Metacello to recognize this, but that's no big deal ... also `cypress://` means that it will be easier to eventually break away from `filetree://` (in the distant future:)